vizro-core/mkdocs.yml

```diff
@@ -98,6 +98,7 @@ markdown_extensions:
   - pymdownx.arithmatex:
       generic: true
   - footnotes
+  - pymdownx.keys
```

Comment:
What is this for?

Reply:
It makes the `++ctrl+shift+p++` render as keyboard keys.